src/event-progress-bar.js

```diff
@@ -0,0 +1,63 @@
+// Update the count down every 1 second
+var x = setInterval( doProgress, 1000 );
```

Comment:
We don't want a very generically named var (`x`) in the global scope.

Reply:
Done.